eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -56,6 +60,15 @@ jobs:
           filePath: "tools/AnalyzeDeps/AnalyzeDeps.ps1"
           arguments: '-PackagesPath "$(Pipeline.Workspace)/packages/*.nupkg" -LockfilePath "$(Build.SourcesDirectory)/eng/Packages.Data.props" -OutPath "$(Build.ArtifactStagingDirectory)/dependencies.html"'
           pwsh: true
+      - pwsh: |
+          $ServiceDir = If ( '${{parameters.ServiceDirectory}}' -eq '*') {'sdk'} Else { '${{parameters.ServiceDirectory}}' }
+          Write-Output "Analyzing Path Lengths..."; $LongestPath = ''; $LongestPathLength = 0
+          Get-ChildItem -Path "$(Build.SourcesDirectory)/$($ServiceDir)" -Recurse -File | ForEach-Object {
+          if ($_.FullName.Length -gt $LongestPathLength) { $LongestPathLength = $_.FullName.Length; $LongestPath = $_.FullName 
+          if ($LongestPathLength -gt 260) { Write-Output "$($LongestPath) : is greater than 260 Character. Reduce path length"; exit 1 }}}
```

Comment:
I think we should limit this to `-ge 260` as windows reserves 1 char for null terminator.